apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/block-iframe-preview.js

```diff
@@ -0,0 +1,176 @@
+/**
+ * External dependencies
+ */
+import { each, filter, get, castArray, debounce } from 'lodash';
+import { createPortal } from 'react-dom';
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+/* eslint-disable import/no-extraneous-dependencies */
+import {
+	useRef,
+	useEffect,
+	useState,
+	useMemo,
+	useReducer,
+	useLayoutEffect,
+	useCallback,
+} from '@wordpress/element';
+import { withSelect } from '@wordpress/data';
+import { BlockEditorProvider, BlockList } from '@wordpress/block-editor';
+import { Disabled } from '@wordpress/components';
+/* eslint-enable import/no-extraneous-dependencies */
+
+const THRESHOLD_RESIZE = 300;
+
+/*
+ * This function will populate the styles from the current document
+ * to the given iFrame head and body DOM references.
+ *
+ * @param {object} els iFrame elements to populate: <head /> and <body />
+ */
+const loadStyles = iFrameDomRefereces => {
+	each( Object.keys( iFrameDomRefereces ), domReference =>
```

Comment:
```suggestion
	each( Object.keys( iFrameDomReferences ), domReference =>
```

`iFrameDomRefereces ` will need search/replace to fix spelling to `iFrameDomReferences`

Reply:
I fixed this.